src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
+
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
+    void add() {
+        assertEquals(7, calculator.add(3, 4));
```

Comment:
1 test is not enough in this case.

We have simple case 3, 4
We have special number - zero, so 10, 0
Also we have large numbers 25004095 5089470
also we have negative numbers 560900 -546

That's a sample approach, when you write unit test, 
cover positive scenario
cover all condition if there are any
cover negative scenario (exception)

And if you could see different subset of arguments, also check corner / edge case.